libbpf-rs/src/query.rs

```diff
@@ -623,6 +623,10 @@ pub struct RawTracepointLinkInfo {
 pub struct TracingLinkInfo {
     /// Attach type of the tracing link.
     pub attach_type: ProgramAttachType,
+    /// Target object ID (`prog_id` for `PROG_EXT`, otherwise btf object id).
```

Comment:
Let's not copy this inscrutable comment verbatim and at least make an attempt to explain what it actually represents, please:
```suggestion
    /// Target object ID (`prog_id` for [`ProgramType::Ext`], otherwise
    /// BTF object id).
```